packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -100,8 +100,14 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call
                        result:(FLTThreadSafeFlutterResult *)result {
   if ([@"availableCameras" isEqualToString:call.method]) {
     if (@available(iOS 10.0, *)) {
+      NSMutableArray *dicscoveryDevices = [NSMutableArray array];
+      [dicscoveryDevices addObject:AVCaptureDeviceTypeBuiltInWideAngleCamera];
+      [dicscoveryDevices addObject:AVCaptureDeviceTypeBuiltInTelephotoCamera];
+      if (@available(iOS 13.0, *)) {
+        [dicscoveryDevices addObject:AVCaptureDeviceTypeBuiltInUltraWideCamera];
+      }
       AVCaptureDeviceDiscoverySession *discoverySession = [AVCaptureDeviceDiscoverySession
-          discoverySessionWithDeviceTypes:@[ AVCaptureDeviceTypeBuiltInWideAngleCamera ]
+          discoverySessionWithDeviceTypes:[NSArray arrayWithArray:dicscoveryDevices]
```

Comment:
can this be just `discoverySessionWIthDeviceTypes: dicscoveryDevices`

Reply:
yes you're right! I simplified that in the tests as well